chargeOclock/Source/DataSources/ClientsDataSource.swift

```diff
@@ -32,9 +27,10 @@ class ClientsDataSource: NSObject, UITableViewDataSource {
 	}
 
 	func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-		let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath) as! ClientTableViewCell
-		cell.client = clients[indexPath.row]
-		cell.updateClient = updateClient
+		let cell = tableView.dequeueReusableCell(withIdentifier: reuseIdentifier, for: indexPath)
```

Comment:
why the separate variable? you could do a:
`if var cell = tableView.deque... {`